content/acra/configuring-maintaining/general-configuration/acra-server/_index.md

```diff
@@ -453,6 +462,146 @@ For additional certificate validation flags, see corresponding pages:
 Should be provided only with `--keystore_encryption_type=<kms_encrypted_master_key|kms_per_client>` flags.
 {{< /hint >}}
 
+
+### HashiCorp Consul
+
+* `--consul_connection_api_string=<url>`
+
+  Connection string (like `https://example.com:8500`) for connecting to HashiCorp Consul for `encryptor_config` loading.
+  If not specified, Consul is not used.
+
+* `--consul_kv_config_path=<schema://host:port>`
+
+  Path to KV storage directory in Consul used to store `encryptor_config`.
+  Default is `acra/encryptor_config`.
+
+* `--consul_tls_client_auth=<mode>`
+
+  Set authentication mode that will be used for TLS connection with Consul.
+
+  * `0` — do not request client certificate, ignore it if received;
+  * `1` — request client certificate, but don't require it;
+  * `2` — expect to receive at least one certificate to continue the handshake;
+  * `3` — don't require client certificate, but validate it if client actually sent it;
+  * `4` — (default) request and validate client certificate.
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--consul_tls_client_ca=<filename>`
+
+  Path to additional CA certificate for Consul certificate validation.
+  Empty by default.
+
+
+* `--consul_tls_client_cert=<filename>`
+
+  Path to server TLS certificate presented to Consul.
```

Comment:
server TLS certificate? Let's elaborate here more, because the parameter is called "client cert".

Reply:
Yes, you are right. My misreading - just copied for @Lagovas PR to have the same TLS flags structure, but it sounds confusing.

@Lagovas I think you need to update it in your PR too.

Reply:
hm, it's acra-server's certificate when acra-server work as client in communication with consul/redis...

Reply:
Maybe just  AcraServer?

Reply:
`Path to AcraServer TLS certificate presented to Consul (AcraServer works as "client" when communicating with Consul).`

I'd put smth like this ^

Reply:
In such case, its make sense to change it in redis-related flag from @Lagovas PR to have a common structure?

Reply:
yes, more clarity is better